src/Microsoft.DotNet.Build.Tasks/Targets/versioning.targets

```diff
@@ -0,0 +1,43 @@
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <UsingTask TaskName="GetDailyBuildNumber" 
```

Comment:
My original thinking was to also perform the updates to AssemblyVersion.cs from this targets file however I didn't flush it out very far.  I will rename this per your suggestion for now, we can always rename it later as the need arises.